service/src/main/java/bio/terra/tanagra/app/authentication/AuthInterceptor.java

```diff
@@ -32,10 +34,19 @@ public class AuthInterceptor implements HandlerInterceptor {
   private static final String OPENAPI_TAG_AUTH_NOT_REQUIRED = "Unauthenticated";
 
   private final AuthenticationConfiguration authenticationConfiguration;
+  private final JwtAccessTokenUtils jwtAccessTokenUtils;
```

Comment:
Instead of adding a new property to this class that's only used for one auth configuration, can you follow the existing pattern of calling a static method in the token decoder utility class? If you want to do some one-time setup (e.g. reading the key file), you can create a singleton instance and store that in the utility class.

Reply:
done